packages/regl-worldview/src/commands/GLText.js

```diff
@@ -53,6 +53,15 @@ type TextMarkerProps = TextMarker & {
   highlightedIndices?: Array<number>,
   highlightColor?: ?Color,
 };
+
+type GLTextProps = {|
+  autoBackgroundColor?: boolean,
+  scaleInvariantFontSize?: number,
+  resolution?: number,
+  alphabet?: string[],
+  textAtlas?: GeneratedAtlas,
+|};
```

Comment:
Can use this inside `Props` below as another `...spread`. No strong feelings either way from me.

Reply:
I agree, but I had to disable flow checks because of the indexed property in `CommonCommandProps`

Reply:
Ah dang, fine either way I think 👍 .